### ISSUE TITLE
Config/Model Checkpoint Pairing

### ISSUE THREAD

**Issue by MorenoLaQuatra** [jishengpeng/WavTokenizer#47]
Hi!

First of all, thank you for your excellent work! I am currently trying to use pre-trained WavTokenizer models and I wanted to confirm the correct pairing between the available configuration files and the model checkpoints before proceeding with my experiments. 

Based on the README and available configuration files, I have made the following assumptions for pairing each config with its corresponding model checkpoint. Could you please confirm if this is correct?

1. **`wavtokenizer_smalldata_frame40_3s_nq1_code4096_dim512_kmeans200_attn.yaml`**  
   - Should refer to models using 40 tokens per second.

2. **`wavtokenizer_smalldata_frame75_3s_nq1_code4096_dim512_kmeans200_attn.yaml`**  
   - Should refer to models using 75 tokens per second.

### Inferred Pairing:

- **`wavtokenizer_smalldata_frame40_3s_nq1_code4096_dim512_kmeans200_attn.yaml`** should be paired with the following models:
  - `WavTokenizer-small-600-24k-4096`
  - `WavTokenizer-large-600-24k-4096`

- **`wavtokenizer_smalldata_frame75_3s_nq1_code4096_dim512_kmeans200_attn.yaml`** should be paired with the following models:
  - `WavTokenizer-small-320-24k-4096`
  - `WavTokenizer-medium-320-24k-4096`
  - `WavTokenizer-large-320-24k-4096`

Could you please confirm if this pairing is correct or if any adjustments are needed? 

Thank you for your help!

**Comment by jishengpeng** [comment]
> Hi!
> 
> First of all, thank you for your excellent work! I am currently trying to use pre-trained WavTokenizer models and I wanted to confirm the correct pairing between the available configuration files and the model checkpoints before proceeding with my experiments.
> 
> Based on the README and available configuration files, I have made the following assumptions for pairing each config with its corresponding model checkpoint. Could you please confirm if this is correct?
> 
> 1. **`wavtokenizer_smalldata_frame40_3s_nq1_code4096_dim512_kmeans200_attn.yaml`**
>    
>    * Should refer to models using 40 tokens per second.
> 2. **`wavtokenizer_smalldata_frame75_3s_nq1_code4096_dim512_kmeans200_attn.yaml`**
>    
>    * Should refer to models using 75 tokens per second.
> 
> ### Inferred Pairing:
> * **`wavtokenizer_smalldata_frame40_3s_nq1_code4096_dim512_kmeans200_attn.yaml`** should be paired with the following models:
>   
>   * `WavTokenizer-small-600-24k-4096`
>   * `WavTokenizer-large-600-24k-4096`
> * **`wavtokenizer_smalldata_frame75_3s_nq1_code4096_dim512_kmeans200_attn.yaml`** should be paired with the following models:
>   
>   * `WavTokenizer-small-320-24k-4096`
>   * `WavTokenizer-medium-320-24k-4096`
>   * `WavTokenizer-large-320-24k-4096`
> 
> Could you please confirm if this pairing is correct or if any adjustments are needed?
> 
> Thank you for your help!

Yes, you are right, if you want to infer only, no adjustments needed.

**Comment by MorenoLaQuatra** [comment]
Thanks! If I can ask, for the large model trained on speech/audio/music the provided checkpoint is [WavTokenizer-large-speech-75token](https://huggingface.co/novateur/WavTokenizer-large-speech-75token). It seems only speech, am I wrong?

**Comment by jishengpeng** [comment]
> Thanks! If I can ask, for the large model trained on speech/audio/music the provided checkpoint is [WavTokenizer-large-speech-75token](https://huggingface.co/novateur/WavTokenizer-large-speech-75token). It seems only speech, am I wrong?

WavTokenizer-Large-unify-40token support speech, audio, music. WavTokenizer-large-speech-75token support speech only up to now.

**Comment by MorenoLaQuatra** [comment]
Thank you. The link [here](https://huggingface.co/novateur/WavTokenizer-large-unify-75token/tree/main) show an empty repo, do you plan to release the model?
